SPECS/pytorch/pytorch.signatures.json

```diff
@@ -0,0 +1,6 @@
+{
+    "Signatures": {
+        "pytorch-1.13.1-submodules.tar.gz": "04966e5e38c3479d1ed6eb5267326eca8c0ee3812b0e444fa026b848829e1660",
+        "pytorch-1.13.1.tar.gz": "c54427bdd371c76176e69d5ef74770b515c8a61e9bc0541bdabafae8d93b6cdf"
```

Comment:
Question: Why generate two separate tarballs for the submodules and pytorch itself? Is it for modularity, like not having to update the whole tarball when a submodule changes? The reason I ask is for future reference / consistency.

Reply:
No strong reason other than a couple of example packages with similar pattern: `ltp` and `influxdb`. I suppose we could combine the submodules with pytorch itself. I'd defer to the next time we touch this spec.

Reply:
Our upgrade automation relies on having a clean tarball from upstream and the `generate_source_tarball.sh` script to download and prepare the submodules. So that's another reason we're keeping these separate.

Also, a clean upstream tarball makes it possible to verify we've got the right file by comparing our signature with the downloaded upstream file.